data/huion-kamvas-12.tablet

```diff
@@ -0,0 +1,42 @@
+# Huion
+# Kamvas 12
+# GS1161
+#
+# Button Map:
+# (A=1, B=2, C=3, ...)
+#
+#    *-----------------------*
+#    |                       |
+#  A |                       |
+#  B |                       |
+#  C |                       |
+#  D |                       |
+#  E |        TABLET         |
+#  F |                       |
+#  G |                       |
+#  H |                       |
+#    |                       |
+#    *-----------------------*
+#
+[Device]
+Name=Huion Kamvas 12
+ModelName=GS1161
+DeviceMatch=usb:256c:006d:HUION Huion Tablet_GS1161 Pad;usb:256c:006d:HUION Huion Tablet_GS1161 Pen
```

Comment:
```suggestion
DeviceMatch=usb|256c|006d|HUION Huion Tablet_GS1161 Pad|usb|256c|006d|HUION Huion Tablet_GS1161 Pen
```
once rebased because #648 changed from `:` to `|`